gradle.properties

```diff
@@ -15,9 +15,9 @@ pravegaKeycloakVersion=0.8.0
 samplesVersion=0.8.0
 
 ### Flink-connector dependencies
-flinkConnectorVersion=0.8.0
-flinkVersion=1.10.2
-flinkMajorMinorVersion=1.10
+flinkConnectorVersion=0.9.0-50.e9d5a98-SNAPSHOT
```

Comment:
I thought we only used released versions in samples. If we make this change, then we will need to make sure to change to the `0.9.0` release of the connector before we cut a samples release, is it right?

Reply:
The target branch of the PR is `dev`, which I think using `snapshot` versions is fine here.
For the release, it is correct to cut the release after the connector release. This is also what we do for previous releases, @RaulGracia . 
There is actually a dependency tree now as for the Pravega ecosystem which causes such order. For example, a chain for the coming 0.9 release is like Pravega -> Pravega schema registry -> Pravega Flink connector -> Pravega samples

Reply:
Make sure to block the release on the update of the version.